src/components/Text/styled.tsx

```diff
@@ -19,13 +20,24 @@ export type Size = typeof SIZES[number];
 export const WEIGHTS = ['light', 'regular', 'bold'] as const;
 export type Weight = typeof WEIGHTS[number];
 
-export const Styled = styled.div<{ color?: string; size: Size; weight?: Weight }>`
+export const Styled = styled.div<{
+  align?: AlignSelfProperty;
```

Comment:
If it is a prop for `align-self`, how about let's just call it `alignSelf`? In case if there is any possibility that we have something like `alignItems` in the future. Let me know if we just need alignSelf here btw

Reply:
Fair enough, will change the prop name. Probably just need `alignSelf` because it's not a flex item, so `alignItems` wouldn't do anything. I could make it a flex item and add it, though.

Reply:
Turns out it is a flex item, so I did add the `alignItems` prop. Thanks for the suggestion!

Reply:
<img width="868" alt="image" src="https://user-images.githubusercontent.com/15721063/100191972-f5f2fc80-2f55-11eb-87ac-a0dddc83312e.png">

Great suggestion. 😄